components/EmotionChart.js

```diff
@@ -0,0 +1,78 @@
+import Plot from "react-plotly.js";
+import styled from "styled-components";
+
+const StyledChartSection = styled.section`
+  background-color: var(--section-background);
+  align-self: center;
+  color: var(--contrast-text);
+  width: fit-content;
+  border-radius: 6px;
+  border: var(--circle-border);
+  margin: 1rem;
+  padding: 0.5rem;
+`;
+
+export default function EmotionChart({
+  xValues,
+  yValues,
+  title,
+  xTitle,
+  yTitle,
+  theme,
+  type,
+  width,
+}) {
+  return (
+    <StyledChartSection>
+      {xValues.length < 2 ? (
+        <p>Sorry, there is not enough data to draw the chart!</p>
+      ) : (
+        <Plot
+          data={[
+            {
+              x: xValues,
+              y: yValues,
+              type: type,
+              mode: "lines+markers",
+              marker: { color: theme.buttonBackground },
+            },
+          ]}
+          config={{ displayModeBar: false, scrollZoom: true }}
+          layout={{
+            font: { color: theme.text },
+            paper_bgcolor: theme.sectionBackground,
+            border_radius: 6,
+            plot_bgcolor: theme.background,
+            xaxis: {
+              gridcolor: theme.text,
+              title: { text: xTitle },
+              font: {
+                size: 15,
+                color: theme.text,
+              },
+              automargin: true,
+            },
+            yaxis: {
+              gridcolor: theme.text,
+              title: { text: yTitle },
+              font: {
+                size: 15,
+                color: theme.text,
+              },
+            },
+            width: width,
+            title: {
+              font: {
+                family: "system-ui",
+                weight: "bold",
+              },
+            },
+            autosize: true,
+            margin: { autoexpand: true, b: 40, l: 30, r: 30, t: 50 },
+            title: title,
```

Comment:
Yeah, and then I made a new definition that has overwritten the former one. This happens when you copy and past from the huge library documentation.

So technically the old one is the one to be deleted, because this code was not executed anyway. So tried the title in bold and deleted that because I do not like it and it was never there before.